app/src/main/java/com/github/se/cyrcle/ui/list/ListScreen.kt

```diff
@@ -339,14 +363,26 @@ fun SpotCard(
                         testTag = "ParkingNoReviews")
                   }
                 }
+
                 Column(horizontalAlignment = Alignment.End) {
-                  Text(
-                      text =
-                          if (distance < 1)
-                              stringResource(R.string.distance_m).format(distance * 1000)
-                          else stringResource(R.string.distance_km).format(distance),
-                      style = MaterialTheme.typography.bodySmall,
-                      testTag = "ParkingDistance")
+                  Row {
+                    // Bearing wrt user's location. This is the orientation wrt north
```

Comment:
Could/Should this be displayed anywhere on the screen in your opinion. Since we're dealing with direction explaining what the "base" is may be a good idea (as someone who never knows where north is haha)

Reply:
I was thinking about it, trust me. But there's just nowhere on the screen where this would make sense and not overcrowd it. I was waiting for suggestions. I think the user can guess that it's respect to north and not their own orientation given that changing the orientation of the phone has no effect (and that the can set a "hardcoded" location thanks to the search bar)

Reply:
I understand your point! I think indeed the tutorial seems like the way to go

Reply:
Happy the solution I'm suggesting works for you. It's out of your PR so I'll go ahead and approve